### DISCUSSION TITLE
Histogram Panel Tooltip

### DISCUSSION THREAD

**Discussion by kdorsel** [grafana/grafana#47848] [Dashboards, Panels & Visualizations]
The Histogram panel should support Tooltips to show the quantity in each bucket when the mouse is hovered over any bucket. For example:
![image](https://user-images.githubusercontent.com/2569415/163824804-081f2e56-2b66-4fc4-a5f8-2838267b0239.png)

The current Histogram Panel does not show tooltips in the settings.
![image](https://user-images.githubusercontent.com/2569415/163824462-bdcda7b7-5fc4-4ea3-aaf8-7dc1a3069a41.png)

However in the documentation is does mention tooltips as being supported. I think the documentation needs to be updated.
https://grafana.com/docs/grafana/latest/visualizations/histogram/

Compated to the graph panel.
![image](https://user-images.githubusercontent.com/2569415/163824533-f00cf8cc-6bad-473d-b571-e024b8d90e9d.png)

**Comment by fabioleite0** [comment]
up!

**Comment by ghost** [comment]
Up!

**Comment by danwalkeruk** [comment]
+1

**Comment by natellium** [comment]
Hey there! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

I will be closing this discussion, but if you’re still interested in this request being implemented please don’t hesitate to open a [feature request](https://github.com/grafana/grafana/issues/new?assignees=&labels=&projects=&template=1-feature_requests.md). Sorry for the inconvenience, and thanks for your contributions!

cc @grafana/dataviz-squad

**Comment by danwalkeruk** [comment]
[Created](https://github.com/grafana/grafana/issues/79375).
